drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -72,6 +74,16 @@ static void _irq_handler(void *arg)
 }
 #endif
 
+static inline void _tigger_send(at86rf2xx_t *dev)
+{
+    netdev_t *netdev = &dev->netdev.netdev;
+    at86rf2xx_tx_exec(dev);
+    dev->status = AT86RF2XX_TRX_STATE__TRAC_INVALID;
+    if (netdev->event_callback && (dev->flags & AT86RF2XX_OPT_TELL_TX_START)) {
+        netdev->event_callback(&dev->netdev.netdev, NETDEV_EVENT_TX_STARTED);
+    }
```

Comment:
```suggestion
    if (netdev->event_callback) {
        netdev->event_callback(&dev->netdev.netdev, NETDEV_EVENT_TX_STARTED);
    }
```

Needs a rebase. The API has changed so that events are reported unconditionally and upper layers just ignore them, if they don't care.